auditlog/models/rule.py

```diff
@@ -362,30 +367,36 @@ def create_logs(self, uid, res_model, res_ids, method,
         http_session_model = self.env['auditlog.http.session']
         for res_id in res_ids:
             model_model = self.env[res_model]
-            name = model_model.browse(res_id).name_get()
+            name = model_model.search([('id', '=', res_id)]).name_get()
             res_name = name and name[0] and name[0][1]
-            vals = {
-                'name': res_name,
-                'model_id': self.pool._auditlog_model_cache[res_model],
-                'res_id': res_id,
-                'method': method,
-                'user_id': uid,
-                'http_request_id': http_request_model.current_http_request(),
-                'http_session_id': http_session_model.current_http_session(),
-            }
-            vals.update(additional_log_values or {})
-            log = log_model.create(vals)
-            diff = DictDiffer(
-                new_values.get(res_id, EMPTY_DICT),
-                old_values.get(res_id, EMPTY_DICT))
-            if method is 'create':
-                self._create_log_line_on_create(log, diff.added(), new_values)
-            elif method is 'read':
-                self._create_log_line_on_read(
-                    log, old_values.get(res_id, EMPTY_DICT).keys(), old_values)
-            elif method is 'write':
-                self._create_log_line_on_write(
-                    log, diff.changed(), old_values, new_values)
+
+            if res_name and res_name != "False":
+                vals = {
+                    'name': res_name,
+                    'model_id': self.pool._auditlog_model_cache[res_model],
+                    'res_id': res_id,
+                    'method': method,
+                    'user_id': uid,
+                    'http_request_id': http_request_model.
+                    current_http_request(),
+                    'http_session_id': http_session_model.
+                    current_http_session(),
+                }
+                vals.update(additional_log_values or {})
+                log = log_model.create(vals)
+                diff = DictDiffer(
+                    new_values.get(res_id, EMPTY_DICT),
+                    old_values.get(res_id, EMPTY_DICT))
+                if method is 'create':
+                    self._create_log_line_on_create(
+                        log, diff.added(), new_values)
+                elif method is 'read':
+                    self._create_log_line_on_read(
+                        log, old_values.get(res_id, EMPTY_DICT).keys(),
+                        old_values)
+                elif method is 'write':
+                    self._create_log_line_on_write(
+                        log, diff.changed(), old_values, new_values)
```

Comment:
If the only change is the condition `if res_name and res_name != "False"`, limit the diff by rewriting it to use the `continue` statement as we are in a `for` loop.
But this change also need some clarification, have you an example of how you encountered a "False" string value?

Reply:
There is, always, a case when creates a register of the action performed, but creates two of them each time, one with (example): res.partned(1,), and the other the same res.partned(1,), but the first the attributes in False, like the name, although it's the object with the correct ID: res.partned(1,).

Reply:
Can you write
```python
if not res_name or res_name == "False":
   continue
```
to avoid an indentation level? It makes the diff very big (and harder to backport too).